src/client/components/search/Search.jsx

```diff
@@ -54,9 +61,11 @@ const Search = ({
   );
 };
 Search.propTypes = {
```

Comment:
The word "search" is in every search prop, and probably needs to be in none of them lol